src/util/i18n.js

```diff
@@ -0,0 +1,35 @@
+import assign from 'react/lib/Object.assign';
+import messages from '../locales';
+var exports = {};
+
+if(global.window) {
```

Comment:
Ah true I forgot that only in donation we have server running this code as well.